src/frontend/src/binder/expr/column.rs

```diff
@@ -37,6 +37,11 @@ impl Binder {
             }
         };
 
+        // Special check for sql udf
```

Comment:
I think this is not easy to understand in the first place. Please be more detailed in the comment here: Specifically, why binding a column need to care about UDF?

Reply:
Indeed, thanks for pointing this out!